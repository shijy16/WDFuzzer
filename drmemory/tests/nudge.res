# **********************************************************
# Copyright (c) 2011-2014 Google, Inc.  All rights reserved.
# Copyright (c) 2009-2010 VMware, Inc.  All rights reserved.
# **********************************************************
#
# Dr. Memory: the memory debugger
#
# This library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation;
# version 2.1 of the License, and no later version.
#
# This library is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Library General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public
# License along with this library; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
#
%OUT_OF_ORDER
# XXX: extra leak due to encoded pointer (PR 482555) is no longer happening on
# my machine!  not sure what's going on
#%if WINDOWS
#LEAK 128 direct bytes + 0 indirect bytes
#crtheap.c:61
#%endif
LEAK 160 direct bytes + 0 indirect bytes
infloop.c:96
LEAK 42 direct bytes + 17 indirect bytes
infloop.c:85
