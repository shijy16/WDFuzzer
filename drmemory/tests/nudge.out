# **********************************************************
# Copyright (c) 2011-2012 Google, Inc.  All rights reserved.
# Copyright (c) 2009-2010 VMware, Inc.  All rights reserved.
# **********************************************************
#
# Dr. Memory: the memory debugger
#
# This library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation;
# version 2.1 of the License, and no later version.
#
# This library is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Library General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public
# License along with this library; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
#
starting
# On Windows, the prefixes for messages from the nudge handler use ~~TID~~
# prefixes due to the injected thread.  On Linux, it's ~~Dr.M~~.  Just match the
# trailing tildes.
# First nudge error report.
~~ ERRORS FOUND:
~~       0 unique,     0 total unaddressable access(es)
~~       0 unique,     0 total uninitialized access(es)
~~       0 unique,     0 total invalid heap argument(s)
~~       0 unique,     0 total warning(s)
~~       2 unique,    21 total,   3259 byte(s) of leak(s)
~~       0 unique,     0 total,      0 byte(s) of possible leak(s)
# Second nudge error report.  We don't match the output of it, just that it was
# here.
~~ ERRORS FOUND:
# On exit error report.  We don't get this on Windows because we use DRkill to
# end infloop.exe.
%if NOSYMS
~~ ERRORS FOUND:
%endif
